src/service/constants.js

```diff
@@ -1 +1 @@
-export const API_URL = process.env.REACT_APP_API_URL;
+export const API_URL = process.env.I_URL;
```

Comment:
`REACT_APP_API_URL` is much more readable than `I_URL` I suspect it's been accidentally deleted